dev-docs/modules/mass.md

```diff
@@ -139,16 +139,46 @@ pbjs.que.push(function() {
 
 ## Integration Example
 
+There are two options to view the integration example:
+
+### Option 1 - Your own development environment
 To view the integration example:
  
-1) in your cli run:
+1) Build Prebid using the following required options
+
+```
+gulp build --modules=ixBidAdapter,mass
+```
+
+2) Use a http server with a valid hostname to access its content. It is not advices to run the bid simulation using localhost or 127.0.0.1
 
 ```
-gulp serve --modules=ixBidAdapter,mass
+http://hostname/integrationExamples/mass/index.html
 ```
 
-2) in your browser, navigate to:
+### Option 2 - Hosted online
+Mass Platform Limited hosts an official integration and demo page that can be accessed using the following link: http://demo.massplatform.com/ix/prebid/
+
+## Testing MASS
+Testing requires valid bids to be returned to Prebid. To assist with this process, we recommend to use the MASS Bid Simulation tool found at https://github.com/massplatform/bidsim. Your Exchange partner might be able to assist you with other specialist tools and browser plugins to achieve similar resuls.
```

Comment:
Typo: >>we recommend to use the MASS Bid Simulation tool<< should be:

we recommend you use the MASS Bid Simulation tool